### PR TITLE
Closes #25. Add oozie authorization and copy files from hdfs through ssh

### LINKED ISSUE #25: Oozie api authorization for integration-tests

### PR DESCRIPTION
**Pull request by madryk**

### CODE REVIEW

File: iis-core/src/test/java/eu/dnetlib/iis/core/SshBasedHdfsFileFetcher.java [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+		} finally {
+			if (sshSession != null) {
+				sshSession.close();
+			}
+		}
+		
+		return exitStatus;
+	}
+	
+	private String appendRemoteSeparatorIfMissing(String directoryPath) {
```
Comment (lukdumi) [inline iis-core/src/test/java/eu/dnetlib/iis/core/SshBasedHdfsFileFetcher.java]: RemoteSeparator suggests that it is a special kind of separator (some remote separator). I would just name this method: appendFilePathSeparatorIfMissing.

File: iis-core/src/test/java/eu/dnetlib/iis/core/SshBasedHdfsFileFetcher.java [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+	//------------------------ PRIVATE --------------------------
+	
+	private void checkIfFileExistsOnHdfs(SSHClient sshClient, String hdfsPath) throws IOException {
+		if (sshExec(sshClient, "hadoop fs -test -e " + hdfsPath, false) != 0) {
+			throw new FileNotFoundException("File " + hdfsPath + " not found on hdfs");
+		}
+	}
+	
+	private void copyFromHdfsOnRemote(SSHClient sshClient, String hdfsSource, String remoteTarget) throws IOException {
+		sshExec(sshClient, "hadoop fs -get " + hdfsSource + " " + remoteTarget);
```
Comment (lukdumi) [inline iis-core/src/test/java/eu/dnetlib/iis/core/SshBasedHdfsFileFetcher.java]: Shouldn't we use some temp folder? Won't it override something?

File: iis-core/src/test/java/eu/dnetlib/iis/core/SshBasedHdfsFileFetcher.java (hunk 1 above)
Comment (madryk) [inline iis-core/src/test/java/eu/dnetlib/iis/core/SshBasedHdfsFileFetcher.java]: ok

Comment (madryk) [inline iis-core/src/test/java/eu/dnetlib/iis/core/SshBasedHdfsFileFetcher.java]: ok

File: iis-core/src/test/java/eu/dnetlib/iis/core/SshBasedHdfsFileFetcher.java (hunk 2 above)
Comment (madryk) [inline iis-core/src/test/java/eu/dnetlib/iis/core/SshBasedHdfsFileFetcher.java]: Well. Some temp folder is good idea.

**Pull request closed without merging**